src/nim_status_client.nim

```diff
@@ -43,7 +43,10 @@ proc mainProc() =
   var node = node.newController(status)
   engine.setRootContextProperty("nodeModel", node.variant)
 
-  var profile = profile.newController(status)
+  proc changeLanguage(locale: string) =
+    engine.setTranslationPackage(fmt"/home/jonathan/dev/nim-status-client/ui/i18n/qml_{locale}.qm")
```

Comment:
???

Reply:
ONLY MY PC WORKS! Muhahaha

Reply:
But seriously, yeah I was tired when I finished the PR so I forgot to change it back to relative haha

Reply:
> ONLY MY PC WORKS! Muhahaha

We'll ship your PC :smiling_imp:

Reply:
Fixed